include/deal.II/lac/linear_operator.h

```diff
@@ -851,6 +841,74 @@ null_operator(const LinearOperator<Range, Domain, Payload> &op)
 }
 
 
+/**
+ * @relatesalso LinearOperator
+ *
+ * Return a LinearOperator that acts as a mean value filter. The vmult()
+ * functions of this matrix filter out mean values of the vector.
```

Comment:
I think it would be clearer to write 'The `vmult()` functions of this matrix subtract the mean value from the vector'.